app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx

```diff
@@ -7,18 +7,18 @@ export const Legend = async ({ lang }: { lang: Language }) => {
   const { t } = await serverTranslation("form-builder", { lang });
   return (
     <div className="my-6">
-      <div className="mb-4 flex text-base italic">
-        <span className="inline-block pr-4">
-          <LogicSectionArrowIcon />
-        </span>
-        <span className="inline-block">{t("logic.legend.page")}</span>
-      </div>
       <div className="text-base italic">
```

Comment:
Looks like `mb-4 flex` was dropped here

We'll need that back to get some spacing

<img width="591" alt="Screenshot 2024-07-31 at 1 26 15 PM" src="https://github.com/user-attachments/assets/1f75caa7-ee34-431a-89a2-2157dd387bbb">

Reply:
Good catch! Thank you!